student_explorer/settings.py

```diff
@@ -353,7 +354,23 @@ def show_debug_toolbar(request):
 
 if DEBUG:
     from debug_toolbar import settings as dt_settings
-    DEBUG_TOOLBAR_PANELS = dt_settings.PANELS_DEFAULTS
+    DEBUG_TOOLBAR_PANELS = dt_settings.PANELS_DEFAULTS + ['cachalot.panels.CachalotPanel',]
     DEBUG_TOOLBAR_CONFIG = {"SHOW_TOOLBAR_CALLBACK" : show_debug_toolbar,}
     INSTALLED_APPS += ('debug_toolbar',)
-    MIDDLEWARE_CLASSES.append('debug_toolbar.middleware.DebugToolbarMiddleware')
\ No newline at end of file
+    MIDDLEWARE_CLASSES.append('debug_toolbar.middleware.DebugToolbarMiddleware')
+
+# Cache time to live
+CACHALOT_TIMEOUT = config("CACHE_TTL", default=60 * 15, cast=int)
```

Comment:
in .env.sample, the default is 60*10, instead of 60*15. Can we make them consistent?

Reply:
Hmm, you're right! I think our local TTL will be something like an hour since this data only changes once a day but better to start off on the low side to see how it's working and edge up for performance.

Reply:
```suggestion
CACHALOT_TIMEOUT = config("CACHE_TTL", default=60 * 10, cast=int)
```